packages/theming/atlas/src/themesource/atlas_core/web/core/widgets/_input.scss

```diff
@@ -86,7 +86,7 @@
 
     // Dropdown input widget
     select.form-control {
-        $arrow: "data:image/svg+xml;utf8,<svg xmlns='http://www.w3.org/2000/svg' width='15px' height='8px'><polyline id='Line' fill='none' stroke='%23555555' points='0.5 0.5 6 6 11.545361 0.5'></polyline></svg>";
+        $arrow: "resources/arrow.svg";
```

Comment:
The attached file is empty, is that intended? 😄

Reply:
@Keraito , @fixed